session/manager_http.go

```diff
@@ -9,6 +9,8 @@ import (
 	"net/url"
 	"time"
 
+	"github.com/ory/x/urlx"
+
```

Comment:
Nitpick: It could be removed to line 21 to reduce fork size

Reply:
Done.